### DISCUSSION TITLE
Source code of gethex.exe, sha256.exe, waitdirchange.exe

### DISCUSSION THREAD

**Discussion by pingwin840** [anic17/Batch-Antivirus#15] [General]
Hello. The source code of these .bat files are open. But where is the source code of those .exe files that come bundled with them?

**Comment by anic17** [comment]
The source code of gethex.exe and sha256.exe is available at the `source` directory on the branch named `source`.

**Comment by pingwin840** [comment]
waitdirchange.exe?

**Reply by anic17** [reply]
I could not find the source code of `waitdirchange.exe`, sorry. I got it from TheBATeam's GitHub.

**Reply by pingwin840** [reply]
Found! https://github.com/TheBATeam/WaitDirChange-by-Frank-P.-Westlake
